src/Unichain.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity >=0.8.0;
+
+library Unichain {
+
+    /******************************************************************************************************************/
+    /*** Token Addresses                                                                                            ***/
+    /******************************************************************************************************************/
+
+    address internal constant SUSDS = 0xA06b10Db9F390990364A3984C04FaDf1c13691b5;
+    address internal constant USDC  = 0x078D782b760474a361dDA0AF3839290b0EF57AD6;
+    address internal constant USDS  = 0x7E10036Acc4B56d4dFCa3b77810356CE52313F9C;
+
+    /******************************************************************************************************************/
+    /*** Bridging Addresses                                                                                         ***/
+    /******************************************************************************************************************/
+
+    address internal constant CCTP_TOKEN_MESSENGER = 0x4e744b28E787c3aD0e810eD65A24461D4ac5a762;
```

Comment:
Verified this address is correct from https://developers.circle.com/stablecoins/evm-smart-contracts